rfcs/048-concepts-rfcs/README.md

```diff
@@ -0,0 +1,45 @@
+# RFC 048: Concepts work plan
+
+**Status:** :construction: Draft :building_construction:
+
+**Last updated:** 29/04/2022
+
+## Context
+
+We are now starting work on introducing "concepts" to the Wellcome digital platform: identifiable entities like subjects, people, organisations, genres, etc. Works can be tagged with concepts, and concepts can be linked to other concepts by (for example) predicates, in this way forming a knowledge graph. There is a future possibility to infer a work's subjects, as well as manual tagging.
```

Comment:
minor: I would reword this last sentence to be more explicit, so we're all on the same page.

> We will start by using concepts that are manually tagged in the source catalogues, and in future we might infer a work's concepts.